src/main/java/com/stevedutch/assignment14/repository/UserRepository.java

```diff
@@ -10,9 +10,9 @@
 public interface UserRepository extends JpaRepository<User, Long> {
 
 	// XXX wiederhole warum dies funktioniert ohne @Query --> weil es Teil von JpaRepository ist :)
-	public User findByUsername(String username);
+	User findByUsername(String username);
 
 	@Query("select u from User u where username = :username")
-	public List<User> findExactlyOneUserByUsername(String username);
```

Comment:
findExactlyOneUserByUsername method should return a User instead of a List of Users since it finds just one user